wagtail/admin/templates/wagtailadmin/shared/main_nav.html

```diff
@@ -1,18 +1,18 @@
 {% load wagtailadmin_tags %}
 {% load i18n %}
-<nav class="nav-main">
+<nav role="navigation" class="nav-main" aria-label="{% trans "Main navigation" %}">
     <ul>
         {{ menu_html }}
 
-        <li class="footer" id="footer">
+        <li class="footer" id="menu-footer">
             <div class="account" id="account-settings" title="{% trans 'Edit your account' %}">
                 <span class="avatar square avatar-on-dark">
                     <img src="{% avatar_url request.user size=50 %}" />
                 </span>
                 <em class="icon icon-arrow-up-after">{{ request.user.first_name|default:request.user.get_username }}</em>
             </div>
 
-            <ul class="footer-submenu">
+            <ul class="footer-submenu" aria-label="Account">
```

Comment:
What is the reasoning for this change? I see the menu items underneath both have their own labels – does it make sense to have this as a labelled section of the menu?

Also should be using `trans` for translations, `{% trans "Account" %}`.